ethereum_history_api/oracles/src/script/generateProof.ts

```diff
@@ -0,0 +1,12 @@
+import { mainnet } from 'viem/chains';
+import { MonorepoCircuit } from '../noir/circuit/circuit.js';
+import { GetStorageProver } from '../noir/circuit/provers/getStorage.js';
+import { HISTORY_API_FIXTURES } from '../fixtures/historyAPIConfig.js';
+
+const { blockNumber, address, storageKeys } = HISTORY_API_FIXTURES.mainnet.paris.usdc_circle;
```

Comment:
Perhaps we can rename this script to `generateTestProof`. 

Just by reading the name of the script I thought I can pass arbitrary params and generate any proof